samples/README.md

```diff
@@ -1,5 +1,7 @@
 # Available Samples
 
+- [](user-interface/predictiveback/src/main/java/com/example/platform/ui/predictiveback/PBHostingActivity.kt):
```

Comment:
Remove this line as it's not part of this PR

Reply:
Resolved